medusa/search/proper.py

```diff
@@ -312,8 +314,8 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
             # if the show is in our list and there hasn't been a proper already added for that particular episode
             # then add it to our list of propers
             if best_result.indexerid != -1 and (
-                best_result.indexerid, best_result.actual_season, best_result.actual_episodes[0]
-            ) not in list(map(operator.attrgetter('indexerid', 'actual_season', 'actual_episode'), final_propers)):
+                best_result.indexerid, best_result.actual_season, best_result.actual_episodes
+            ) not in list(map(operator.attrgetter('indexerid', 'actual_season', 'actual_episodes'), final_propers)):
```

Comment:
^ This was the reason for muli-episodes repeated downloads.
For a multi-ep:
`actual_episode = None`
while `actual_episodes[0]` would be the first episode of the multi-episode release.

Reply:
Wow, that seems like a bad design.
Not for this PR, but I think `actual_episode` should just point to the first item in `actual_episodes`.
To make sure it's always correct though, it should raise an error if there are multiple episodes in `actual_episodes`.